react/src/components/KeypairResourcePolicySettingModal.tsx

```diff
@@ -367,9 +376,10 @@ const KeypairResourcePolicySettingModal: React.FC<
                             },
                           },
                         ]}
+                        style={{ margin: 0, width: '100%' }}
```

Comment:
Is there any reason to setting this style?

Reply:
I want to match the width with below input fields.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/e480fec5-998a-4429-9d23-335cae01dab6.png)

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/ceba0822-22ad-425a-ad0a-54e2a02c9f0a.png)

The `FormItemWithUnlimited` has `style={{ margin: 0 }}` and it can be override by other components. So, if I just set `style={{ width: 0 }}`, the checkbox margin is too wide.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/e17e0397-3f2d-44fc-a487-b16368f5e161.png)